xpk.py

```diff
@@ -1889,11 +1932,22 @@ def workload_create(args) -> int:
     xpk_print(f'Create Workload request returned ERROR {return_code}')
     xpk_exit(return_code)
 
+  # Get GKE outlier dashboard
+  outlier_dashboard_id = get_gke_outlier_dashboard(args)
+
   xpk_print(
       'Follow your workload here:'
       # pylint: disable=line-too-long
       f' https://console.cloud.google.com/kubernetes/service/{zone_to_region(args.zone)}/{args.cluster}/default/{args.workload}/details?project={args.project}'
   )
+
+  if outlier_dashboard_id is not None:
+    xpk_print(
+        'Check statistics and outlier mode of GKE metrics for your workload here:'
+        # pylint: disable=line-too-long
+        f'http://console.cloud.google.com/monitoring/dashboards/builder/{outlier_dashboard_id}?project={args.project}'
```

Comment:
Is it possible to attach the clusterName, jobName, and podName filters to the link here too?

Reply:
Yeah, great idea. I can add the cluster name to the link. However, it is not ideal to add a pod name filter as there can be multiple pods in a cluster.